lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -108,6 +108,14 @@
 class BezierPath(Line2D):
 
     def __init__(self, path, *args, **kwargs):
+        """
+        Parameters
+        ----------
+        path : path.Path
+            The path to draw.
+        **kwargs
+            All remaining keyword arguments are passed to `Line2D`.
```

Comment:
This probably needs to be .Line2D with a dot?

Reply:
I forgot to test the doc build (only did a REPL check) after fixing the deprecation decorator, but strangely, it does not seem to warn about this either way. I guess I need to update my conda environment, but we might want to bump minimum sphinx requirements as well, so this is more consistent.